cli/ip_address.rs

```diff
@@ -0,0 +1,142 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+use regex::Regex;
+
+/// Check IPv4 address
+pub fn is_ipv4(ip_addr: &str) -> bool {
+  lazy_static! {
+    static ref RE: Regex = Regex::new(
+      format!(
+        r"^(?:{ipv4_part}\.){{3}}{ipv4_part}$",
+        ipv4_part = "(?:[0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])"
+      )
+      .as_ref()
+    )
+    .unwrap();
+  }
+
+  if ip_addr.is_empty() {
+    return false;
+  }
+
+  RE.is_match(ip_addr)
+}
+
+/// Check IPv6 address
+pub fn is_ipv6(ip_addr: &str) -> bool {
```

Comment:
I don't think deno should maintain this type of regex for matching IPs.  I think we could get most of this by using the implementation baked into rust: `std::net::IpAddr`  

`let ip: IpAddr = "::1".parse();`

Reply:
Done.